app/main.py

```diff
@@ -39,10 +45,13 @@ def process(self, encrypted_jwt, tx_id=None):
 
         try:
             decrypted_payload = self._decrypt(encrypted_jwt, tx_id)
+            logger.debug("Decrypted message", tx_id=tx_id)
 
             decoded_contents, file_name = self._extract_file(decrypted_payload, tx_id)
+            logger.debug("Extract file", tx_id=tx_id)
 
             self._send_to_ftp(decoded_contents, file_name, tx_id)
+            logger.debug("Sent to FTP", tx_id=tx_id)
```

Comment:
Sent what to ftp? Should log that it's a file you're sending and the name of the file.